Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+
+## Explanation of why OpenHW should do this project
+
+Like RI5CY, the ARIANE core donated by ETH Zürich and U. Bologna was at the heart of the OpenHW Group creation. We need an application processor core in our portfolio.
+
+Also, there are very few FPGA technology-independent softcores and CV32A6 is an alternative to proprietary cores.
+
+## Industry landscape: description of competing, alternative, or related efforts in the industry
+
+On the core side, for ASIC targets, these are the most comparable competitors (pipeline length, MMU, single issue...):
+- ARM: Cortex-A5
+- SiFive: U54
+- CHIPS Alliance&nbsp;/ Western Digital: CVA6 between the smaller SweRV  EL2 and the larger EH1.
+- ANDES: A25 (32-bit) and AX25 (64-bit)
+- Frontgrade Gaisler: NOEL-V
+
+On the FPGA side, CV32A6 is a technology-independent alternative to these proprietary cores:
+- Xilinx: Microblaze
+- MicroChip: Mi-V
+- Intel: Nios-II, Nios-V
+
+On the tool side, CHIPS Alliance has plans to make their tools open-source
+([link](https://semiengineering.com/components-for-open-source-verification/), 5<sup>th</sup> paragraph).
+
+### Market differentiators
+
+To differentiate from the competition, marketing can stress:
+- Source code written in **SystemVerilog**, a widely accepted language;
+- **Open-source** availability of the cores;
+- Open-source availability of verification artefacts, which is a great step towards certification for security- and safety-critical applications;
+- The availability of a family of **technology-independent cores optimized for ASIC and FPGA targets**;
+- The ability to **extend the instruction** set thanks to the CV-X-IF interface;
+- The **SW ecosystem** running on CVA6, with FreeRTOS and Linux already demonstrated;
+- The ability to scale to SMP multi/many-core CPUs thanks to the OpenPiton framework;
+- The permissive licence scheme that allows the integration in open-source or closed-source projects or the addition of a "secret sauce";
+- The low exposition to export control
+([OpenHW Group Membership Agreement](https://www.openhwgroup.org/membership/openhw-group-membership-agreement-2019-10-16.pdf), section 4.1).
+
+## External dependencies
+
+The project relies on:
+- Related and sister OpenHW projects (joint with other cores): LLVM, FreeRTOS, core-v-verif, CV-X-IF specification, OpenPiton...
+- Open-source software: GCC, GDB, LLVM, Linux (Yocto, BuildRoot), OpenSBI, UBoot, BBL...
+- Open-source verification: Google Riscv-dv, Spike, [riscv-arch-test](https://github.com/riscv-non-isa/riscv-arch-test)
+- Open-source hardware: fpnew (ETH Zürich), first verified in CV32E40Pv2
+- Eclipse Foundation, GitHub
+- Digilent Genesys 2 board
+- Linter: verible
+- Simulators: Verilator (open-source), Siemens Questa, Synopsys VCS, Cadence Xcelium
+- Synthesis: Vivado (AMD, ex Xilinx), Libero (Microchip), Quartus (Intel, ex Altera), Synopsys Design Compiler (ASIC)
+
```

Comment:
The point being made here isn't that clear. On the one hand we're saying that certain tools must be open-source and on the other hand we're listing commercial tools as dependencies. The real-world is that OpenHW projects rely on commercial design tools where appropriate. We've decided to also create open-source tools such as Spike model for CVA6.

Reply:
We do not want to be bound to a specific vendor. There are several providers for  simulators and synthesis tools.